pdr_backend/lake/table.py

```diff
@@ -18,17 +18,21 @@
 
 
 class TableType(Enum):
-    NORMAL = "NORMAL"
+    PROD = "PROD"
+    NEW_EVENTS = "NEW_EVENTS"
+    UPDATE_EVENTS = "UPDATE_EVENTS"
     TEMP = "TEMP"
-    ETL = "ETL"
+    TEMP_UPDATE = "TEMP_UPDATE"
 
 
 @enforce_types
 def is_etl_table(table_name: str) -> bool:
     table_name = table_name.removeprefix("_")
```

Comment:
I'm thinking we can convert this into a loop.
for prefix in [ list of prefixes]: table_name = table_name.removeprefix(prefix)

Reply:
done